src/platforms/common/index.mdx

```diff
@@ -0,0 +1,62 @@
+On this page, we get you up and running with Sentry's SDK, so that it will automatically report errors and exceptions in your application.
+
+<Note>
+<markdown>
+
+If you don't already have an account and Sentry project established, head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+
+</markdown>
+</Note>
+
+<PlatformContent includePath="framework-list" />
+
+## Install
+
+Sentry captures data by using an SDK within your application’s runtime:
+
+<PlatformContent includePath="getting-started-install" />
+
+## Initialize
+
+`init` Sentry as soon as possible in your app. Once this is done, all unhandled exceptions are automatically captured by Sentry.
+
+<PlatformContent includePath="getting-started-config" />
+
+<Alert level="note" title="Note"><markdown>
+
+We'll automatically assign you a Data Source Name (DSN), which looks like a standard URL. It's required by Sentry and configures the protocol, public key, server address, and project identifier. If you forget it, view _Settings -> Projects -> Client Keys (DSN)_ in sentry.io.
+
+</markdown></Alert>
+
+## Test
```

Comment:
should this be Test or Verify?

Reply:
It can be either. It's a test error, or one that verifies. I'm happy with either.